src/dso_api/settings.py

```diff
@@ -68,6 +89,31 @@
     "dso_api.dynamic_api.middleware.TemporalDatasetMiddleware",
     "dso_api.dynamic_api.middleware.RequestAuditLoggingMiddleware",
 ]
+connection_string: Optional[str] = None
```

Comment:
Maybe it would be better to move whole block down below `LOGGING` definition, then we would not need 2 identical `if CLOUD_ENV.is_azure():` statements and probably can get rid of temporary variable `connection_string`.

Reply:
Sure no problem. Though the downside of that approach is that the middleware configuration will then happen at two separate locations in `settings.py`: one where `MIDDLEWARE` is defined, and the other one some time after the logging configuration

Reply:
That's okay, because additional MIDDLEWARE is added conditionally.

Reply:
done